### PR TITLE
Tech plan entity

### PR DESCRIPTION
**Pull request by Marcellinom**
## how to view and edit:
- install plugin puml di intellij
- ~liat di puml editor online (not recomended, biasanya cropnya ga bener)~ 
DONT DO THIS. udah banyak diagramnya sekarang, pakailah option 1

## Picture Below Shows only the Entity Relation Diagram
- Class Relation Diagram
image

### CODE REVIEW

**Comment by zahrafayya** [comment]
the class diagram is so neat!

File: src/Entity/EntityClassDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
++ getNextPosition(): void
++ update(): void
++ draw():void @override
++ getter()
+--ENUMS--
+int IDLE = 0
+int WALKING = 1
+int JUMPING = 2
+int FALLING = 3
+int GLIDING = 4
```
Comment (ryohilmi) [inline src/Entity/EntityClassDiagram.puml:22]: I think you can just remove the gliding state, the player doesn't have any wings

Comment (Marcellinom) [reply src/Entity/EntityClassDiagram.puml:22]: working on it!

**Comment by Marcellinom** [comment]
TODO: change range atk type Peebo

File: src/Entity/EntityClassDiagram.puml (hunk 1 above)
Comment (Marcellinom) [reply src/Entity/EntityClassDiagram.puml]: resolved

**Review by zahrafayya — APPROVED** [review]
looks good

**Pull request merged**